OrderCloud.Integrations.Payment.CardConnect/CardConnectService.cs

```diff
@@ -0,0 +1,43 @@
+﻿using OrderCloud.Catalyst;
+using OrderCloud.Integrations.Payment.CardConnect;
+using OrderCloud.Integrations.Payment.CardConnect.Mappers;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace OrderCloud.Integrations.Payment.BlueSnap
```

Comment:
Shouldn't this be a CardConnect namespace?

Reply:
Yes, thank you - updated.